zk/datastream/types/l2block_proto.go

```diff
@@ -0,0 +1,61 @@
+package types
+
+import (
+	"github.com/ledgerwatch/erigon/zk/datastream/proto/github.com/0xPolygonHermez/zkevm-node/state/datastream"
+	"google.golang.org/protobuf/proto"
+	libcommon "github.com/gateway-fm/cdk-erigon-lib/common"
+)
+
+type L2BlockProto struct {
+	*datastream.L2Block
+}
+
+type FullL2Block struct {
+	BatchNumber     uint64
+	L2BlockNumber   uint64
+	Timestamp       int64
+	DeltaTimestamp  uint32
+	L1InfoTreeIndex uint32
+	GlobalExitRoot  libcommon.Hash
+	Coinbase        libcommon.Address
+	ForkId          uint64
+	ChainId         uint64
+	L1BlockHash     libcommon.Hash
+	L2Blockhash     libcommon.Hash
+	StateRoot       libcommon.Hash
+	L2Txs           []L2Transaction
+	ParentHash      libcommon.Hash
+	BatchEnd        bool
```

Comment:
These 2 properties are probably 'hacks' as I just need them out without having the caller have knowledge of batchEnd and batchStart